handlers/BaseHandler.cfc

```diff
@@ -8,6 +8,40 @@
 * to produce RESTFul responses.
 */
 component extends="coldbox.system.EventHandler"{
+	property name="APIResponse" inject="Response";
```

Comment:
Injecting the APIResponse like this actually makes the response a singleton and not a request based object.